src/components/event-delegation.js

```diff
@@ -5,7 +5,7 @@ var getMarkoPropsFromEl = componentsUtil.___getMarkoPropsFromEl;
 
 // We make our best effort to allow multiple marko runtimes to be loaded in the
 // same window. Each marko runtime will get its own unique runtime ID.
-var listenersAttachedKey = "$MED" + runtimeId;
+var listenersAttachedKey = "$MDE" + runtimeId;
```

Comment:
This is also a fix so that the listeners attach to a new runtimeId in the newest Marko version. This makes the newest marko version compatible with all previous ones, but doesn't allow 4.5> to be compatible with 4.4